au4/src/au3wrap/au3project.cpp

```diff
@@ -83,15 +83,13 @@ bool Au3Project::load(const muse::io::path_t& filePath)
     return true;
 }
 
-bool Au3Project::save(const muse::io::path_t& filePath, const bool fromSaveAs)
+bool Au3Project::save(const muse::io::path_t& filePath)
 {
-    //! TODO AU4
-    // auto& projectFileIO = ProjectFileIO::Get(m_data->projectRef());
-    // std::string sstr = filePath.toStdString();
-    // FilePath fileName = wxString::FromUTF8(sstr.c_str(), sstr.size());
-    // TrackList& tracks = TrackList::Get(m_data->projectRef());
-    // bool success = projectFileIO.SaveProject(fileName, nullptr /* m_lastSavedTracks*/);
-    return false;
+    auto& projectFileIO = ProjectFileIO::Get(m_data->projectRef());
+    std::string sstr = filePath.toStdString();
+    FilePath fileName = wxString::FromUTF8(sstr.c_str(), sstr.size());
```

Comment:
By the way, we have https://github.com/audacity/audacity/blob/master/au4/src/au3wrap/wxtypes_convert.h
It can be improved...

Reply:
changed